lib/collection/src/operations/snapshot_ops.rs

```diff
@@ -65,11 +67,14 @@ pub async fn get_snapshot_description(path: &Path) -> CollectionResult<SnapshotD
                 NaiveDateTime::from_timestamp_opt(duration.as_secs() as i64, 0).unwrap()
             })
     });
+
+    let checksum = crate::common::sha_256::hash_file(path).await?;
```

Comment:
this will be potentially a very slow operation, especially considering that we would need to do it on the list of snapshots. I don't think we can afford to read several GBs of data on each request

Reply:
Okay, I wasn't aware of how big the snapshots might be. In this case, the checksum could be calculated after the snapshot is ready. Is there already some place, where the metadata about the snapshot it is stored, or should I add some file that could store this checksum?

Reply:
This is definitely a valid point to consider. If the use-case would be to mostly do it just for a sanity check on the user side, something lighter would definitely do the job as well.